src/mscorlib/model.xml

```diff
@@ -1825,6 +1825,22 @@
       <Member Name="FailFast(System.String,System.Exception)"  />
       <Member Name="Exit(System.Int32)" />
     </Type>
+    <Type Name="Internal.Runtime.Augments.EnvironmentAugments">
```

Comment:
I agree that it needs the `Environment` need stay for now until corefx is updated. Everything will break otherwise.

> I'm not sure we can remove the existing Environment exports, in case we need to support an existing System.Runtime.Extensions.dll working with a newer System.Private.Corelib.dll... not sure if that's a supported scenario or not.

I have said number of times that we need to be a plan where only latest CoreCLR is guaranteed to work with latest  System.Runtime.Extensions and friends (assemblies that depend on internal contracts). The matrix of different combinations will go through the roof otherwise, and we are unlikely to ensure that all combinations work anyway.